plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraServer.java

```diff
@@ -13,165 +13,17 @@
  */
 package io.trino.plugin.cassandra;
 
-import com.datastax.oss.driver.api.core.CqlSession;
```

Comment:
Please preserve the git history of this class.

Reply:
i've tried to move to preserve git history, but during rebase taking in account that same file name created it is not preserved.

Reply:
You can add a preparatory commit.